src/klein/_plating.py

```diff
@@ -161,11 +163,45 @@ def _elementify(self, instance, to_fill_with):
                              boundInstance=instance,
                              presentationSlots=self._presentationSlots)
 
+    @attr.s
+    class _Widget(object):
+        """
+        Implementation of L{Plating.widgeted}.  This is a L{callable}
+        descriptor that records the instance to which its wrapped
+        function is bound, if any.  Its L{widget} method then passes
+        that instance or L{None} and the result of invoking the
+        function (or now bound method) to the creating L{Plating}
+        instance's L{Plating._elementify} to construct a
+        L{PlatedElement}.
+        """
+        _plating = attr.ib()
+        _function = attr.ib()
+        _instance = attr.ib()
+
+        def __call__(self, *args, **kwargs):
+            return self._function(*args, **kwargs)
+
+        def __get__(self, obj, type_=None):
```

Comment:
Can we obj, type ➜ instance, owner to match __get__signature in the python docs?